substrate/frame/recovery/Cargo.toml

```diff
@@ -29,13 +29,5 @@ runtime-benchmarks = [
 	"frame/runtime-benchmarks",
 	"pallet-balances/runtime-benchmarks",
 ]
-std = [
-	"codec/std",
-	"frame/std",
-	"pallet-balances/std",
-	"scale-info/std",
-]
-try-runtime = [
-	"frame/try-runtime",
-	"pallet-balances/try-runtime",
-]
+std = ["codec/std", "frame/std", "pallet-balances/std", "scale-info/std"]
```

Comment:
I think something is wrong here, taplo should not format them like this.  
Maybe the taplo version is old?

Reply:
Actually, it uses: https://github.com/paritytech/polkadot-sdk/actions/runs/14912508983/job/41890169289?pr=8466
```
taplo 0.8.1
```
For fellows, we are using [0.9.3](https://github.com/polkadot-fellows/runtimes/blob/main/.github/env#L3)

@alvicsam Could you please point me, where taplo version is defined for polkadot-sdk, I cannot find it in polkadot-sdk, somewhere here?
```
IMAGE="docker.io/paritytech/ci-unified:bullseye-1.84.1-2025-01-28-v202502131220"
```
Would it make sense to externalize TAPLO_VERSION like we have for fellows?

Reply:
> Maybe also worth to update the taplo, i think the CI used to run a quite old version.

ok, thank you, let's wait for taplo upgrade https://github.com/paritytech/dockerfiles/issues/17